src/serializers/encoder/v16/encode_graph_entities.c

```
@@ -0,0 +1,505 @@
+
+static void _RdbSaveEntity
+(
+	SerializerIO rdb,
+	const GraphEntity *e
+) {
+	// Format:
+	// #attributes N
+	// (name, value type, value) X N 
+
+	const AttributeSet set = GraphEntity_GetAttributes(e);
+	uint16_t attr_count = AttributeSet_Count(set);
```

cppcheck

[error] Common realloc mistake

(memleakOnRealloc)